jmqtt-broker/src/main/java/org/jmqtt/broker/store/redis/support/RedisReplyUtils.java

```diff
@@ -0,0 +1,7 @@
+package org.jmqtt.broker.store.redis.support;
+
+public class RedisReplyUtils {
+    public static Boolean isOk(String reply){
+        return reply !=null && reply.startsWith("OK");
```

Comment:
I have not tested this logic yet, please do not merge until I remove this comment.